lib/disk/modules/AzureCommon.rb

```diff
@@ -0,0 +1,95 @@
+require_relative '../MiqDisk'
+require 'ostruct'
+
+module AzureCommon
```

Comment:
@@roliveri good idea.  BTW I still have some code climate issues to fix up, and for some reason the math in the spec tests is off.  I will work with that in the morning.